commit b757e2a5dafc44cf3b03051213df79daa3caf5b9
Author: notro <notro@tronnes.org>
Date:   Fri Jul 18 18:15:57 2014 +0200

    BCM2708: DT: change 'axi' nodename to 'soc'
    
    Change DT node named 'axi' to 'soc' so it matches ARCH_BCM2835.
    The VC4 bootloader fills in certain properties in the 'axi' subtree,
    but since this is part of an upstreaming effort, the name is changed.
    
    Signed-off-by: Noralf Tronnes notro@tronnes.org

diff --git a/arch/arm/boot/dts/bcm2708.dtsi b/arch/arm/boot/dts/bcm2708.dtsi
index 3f884b3..e02e67b 100644
--- a/arch/arm/boot/dts/bcm2708.dtsi
+++ b/arch/arm/boot/dts/bcm2708.dtsi
@@ -14,7 +14,7 @@
 		bootargs = "console=ttyAMA0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 ";
 	};
 
-	axi {
+	soc {
 		compatible = "simple-bus";
 		#address-cells = <1>;
 		#size-cells = <1>;
